app/src/main/res/layout-land/ongoing_story_card.xml

```diff
@@ -58,7 +58,6 @@
         <TextView
           android:id="@+id/story_name_text_view"
           style="@style/Body"
-          android:layout_width="match_parent"
```

Comment:
Why was this removed? I think this is different from `styles.xml` file as it is not wrap_content